test/goss/README.MD

```diff
@@ -0,0 +1,44 @@
+# Testing the Container with DGOSS
+
+[GOSS](https://github.com/aelsabbahy/goss) is a *server validation tool* and together with [DGOSS](https://github.com/aelsabbahy/goss/tree/master/extras/dgoss) it can be used to validate Docker conteiners.
```

Comment:
```suggestion
[GOSS](https://github.com/aelsabbahy/goss) is a *server validation tool* and together with [DGOSS](https://github.com/aelsabbahy/goss/tree/master/extras/dgoss) it can be used to validate Docker containers.
```